client/src/components/participant-form/EmailSubmissionForm.js

```diff
@@ -51,11 +52,39 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+const addEmailToWhitelist = async (email, openToast) => {
+  const resp = await fetch(`${API_URL}/api/v1/participants/waitlist`, {
+    headers: { Accept: 'application/json', 'Content-type': 'application/json' },
+    body: JSON.stringify({ email }),
+    method: 'POST',
+  });
+  if (resp.ok) {
+    openToast({
+      status: ToastStatus.Success,
+      message:
+        'Your email has been added to the list. You will be notified when submissions are open.',
+    });
+  } else if (resp.status === 400) {
```

Comment:
409 is probably better here

Reply:
Should throw an error in the else for non 200/400 errors so the catch later on is triggered?